clients/src/test/java/org/apache/kafka/clients/producer/internals/ProducerBatchTest.java

```diff
@@ -72,7 +72,11 @@ public void testBatchAbort() throws Exception {
 
         // subsequent completion should be ignored
         batch.done(500L, 2342342341L, null);
-        batch.done(-1, -1, new KafkaException());
+        try {
```

Comment:
This test will pass whether line 76 throws IllegalStateException or not. Should we add a fail() statement after line 76?